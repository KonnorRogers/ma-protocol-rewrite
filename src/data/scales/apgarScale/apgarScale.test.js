import * as React from "react"
import { render } from "@testing-library/react"
import ApgarScale from "./index"

describe("ApgarScale()", () => {
  test("Should render without error", () => {
    const { asFragment } = render(<ApgarScale />)

    expect(asFragment()).toMatchSnapshot()
  })
})
