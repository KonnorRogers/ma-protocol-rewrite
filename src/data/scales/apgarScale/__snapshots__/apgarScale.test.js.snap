// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApgarScale() Should render without error 1`] = `
<DocumentFragment>
  <h3
    id="apgar-scale"
    style="text-align: center; margin-top: 1.5rem;"
  >
    <a
      href="#apgar-scale"
    >
      APGAR Scale
    </a>
  </h3>
  .emotion-50 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  border: 1px solid white;
  min-width: 400px;
}

.emotion-50 div:last-of-type {
  border-bottom: none;
}

.emotion-4 {
  font-weight: bold;
  font-size: 1.15rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  text-align: center;
  width: 100%;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border-bottom: 1px solid white;
  font-size: 0.75rem;
}

.emotion-4 div:last-of-type {
  border-right: none;
}

@media only screen and (min-width:640px) {
  .emotion-4 {
    font-size: 0.85rem;
  }
}

@media only screen and (min-width:768px) {
  .emotion-4 {
    font-size: 1.1rem;
  }
}

.emotion-0 {
  width: 25%;
  border-right: 1px solid white;
  padding: 0.5rem;
  text-align: center;
}

.emotion-13 {
  font-weight: normal;
  font-size: initial;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  text-align: center;
  width: 100%;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border-bottom: 1px solid black;
}

.emotion-13 div:last-of-type {
  border-right: none;
}

.emotion-6 {
  width: 25%;
  border-right: 1px solid black;
  padding: 0.5rem;
  text-align: center;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
}

.emotion-50 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  border: 1px solid white;
  min-width: 400px;
}

.emotion-50 div:last-of-type {
  border-bottom: none;
}

.emotion-4 {
  font-weight: bold;
  font-size: 1.15rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  text-align: center;
  width: 100%;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border-bottom: 1px solid white;
  font-size: 0.75rem;
}

.emotion-4 div:last-of-type {
  border-right: none;
}

@media only screen and (min-width:640px) {
  .emotion-4 {
    font-size: 0.85rem;
  }
}

@media only screen and (min-width:768px) {
  .emotion-4 {
    font-size: 1.1rem;
  }
}

.emotion-0 {
  width: 25%;
  border-right: 1px solid white;
  padding: 0.5rem;
  text-align: center;
}

.emotion-13 {
  font-weight: normal;
  font-size: initial;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  text-align: center;
  width: 100%;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  border-bottom: 1px solid black;
}

.emotion-13 div:last-of-type {
  border-right: none;
}

.emotion-6 {
  width: 25%;
  border-right: 1px solid black;
  padding: 0.5rem;
  text-align: center;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
}

<div
    class="emotion-50"
  >
    <div
      class="emotion-4"
    >
      <div
        class="emotion-0"
      >
        Feature Evaluated
      </div>
      <div
        class="emotion-0"
      >
        2 Points
      </div>
      <div
        class="emotion-0"
      >
        1 Point
      </div>
      <div
        class="emotion-0"
      >
        0 Points
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
        style="background-color: rgb(255, 227, 227); color: black;"
      >
        <div
          class="emotion-5"
        >
          <span>
            <strong>
              A
            </strong>
            ctivity (Muscle Tone)
          </span>
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(255, 227, 227); color: black;"
      >
        <div
          class="emotion-5"
        >
          Active Movement
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(255, 227, 227); color: black;"
      >
        <div
          class="emotion-5"
        >
          Arms and legs flexed (Weak, some movement)
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(255, 227, 227); color: black;"
      >
        <div
          class="emotion-5"
        >
          Limp or flaccid
        </div>
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
        style="background-color: rgb(221, 214, 229); color: black;"
      >
        <div
          class="emotion-5"
        >
          <span>
            <strong>
              P
            </strong>
            ulse
          </span>
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(221, 214, 229); color: black;"
      >
        <div
          class="emotion-5"
        >
          Over 100 bpm
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(221, 214, 229); color: black;"
      >
        <div
          class="emotion-5"
        >
          Below 100bpm
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(221, 214, 229); color: black;"
      >
        <div
          class="emotion-5"
        >
          Absent
        </div>
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
        style="background-color: rgb(215, 227, 191); color: black;"
      >
        <div
          class="emotion-5"
        >
          <span>
            <strong>
              G
            </strong>
            rimace (Irritability / Reflexes)
          </span>
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(215, 227, 191); color: black;"
      >
        <div
          class="emotion-5"
        >
          Cry, sneeze, cough, active movement
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(215, 227, 191); color: black;"
      >
        <div
          class="emotion-5"
        >
          Grimace (some flexion of extremities)
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(215, 227, 191); color: black;"
      >
        <div
          class="emotion-5"
        >
          No Reflexes
        </div>
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
        style="background-color: rgb(191, 207, 226); color: black;"
      >
        <div
          class="emotion-5"
        >
          <span>
            <strong>
              A
            </strong>
            ppearance (Skin Color)
          </span>
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(191, 207, 226); color: black;"
      >
        <div
          class="emotion-5"
        >
          Completely pink
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(191, 207, 226); color: black;"
      >
        <div
          class="emotion-5"
        >
          Body pink, extremities blue
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(191, 207, 226); color: black;"
      >
        <div
          class="emotion-5"
        >
          Blue, pale
        </div>
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
        style="background-color: rgb(250, 250, 175); color: black;"
      >
        <div
          class="emotion-5"
        >
          <span>
            <strong>
              R
            </strong>
            espiration
          </span>
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(250, 250, 175); color: black;"
      >
        <div
          class="emotion-5"
        >
          Vigorous cry, Full breaths
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(250, 250, 175); color: black;"
      >
        <div
          class="emotion-5"
        >
          Slow, irregular, or gasping breaths, weak cry
        </div>
      </div>
      <div
        class="emotion-6"
        style="background-color: rgb(250, 250, 175); color: black;"
      >
        <div
          class="emotion-5"
        >
          Absent
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
